kv-tokens/configs/tailwind.config.js

```diff
@@ -15,9 +15,15 @@ const {
 	radii,
 } = designtokens;
 
+const remCalc = (px) => `${px / 16}rem`;
+
 module.exports = {
 	purge: [],
 	darkMode: false, // or 'media' or 'class'
+	corePlugins: {
+		fontSize: false,
```

Comment:
This disables tailwind from generating it's default .font-sm, .font-lg, etc. classes.

Reply:
I suspect there will be some follow on work to ensure proper differentiation across the kitchen sink of elements...